go/runtime/client/tests/tester.go

```diff
@@ -229,6 +229,11 @@ func testQuery(
 		Data:      []byte("test checktx request"),
 	})
 	require.NoError(t, err, "CheckTx")
+
+	// Get the number of unconfirmed transactions.
+	utxs, err := c.GetUnconfirmedTransactions(ctx, runtimeID)
+	require.NoError(t, err, "GetUnconfirmedTransactions")
+	require.True(t, len(utxs) == 0)
```

Comment:
Is there a test where this is > 0?

Reply:
No. But the consensus `GetUnconfirmedTransactions` test is similar :P

I did initially want to make this nicer, but the mock runtime host that's used in unit tests sucks and doing this in the e2e tests at exactly the right time to make it deterministic is super-hard. So I think like with the consensus method that does the same, the test for this should be in the block explorer or wherever they plan on using this.